.travis.yml

```diff
@@ -25,7 +25,7 @@ install:
   - chmod +x coveralls.phar
 
 script:
-  - phpdbg -qrr vendor/bin/phpunit --coverage-text --coverage-clover build/logs/clover.xml
```

Comment:
Disabling the GC is not really a solution. Maybe we should use xdebug again?

Reply:
@kelunik Can you explain please about xdebug ?

Reply:
Xdebug caused compatibility issues on travis in the past, but those may have been resolved. Remove the `before_install` section of `.travis.yml` and remove the portions of this line preceding `vendor/bin/phpunit …`.

Reply:
I did some investigation. Segmentation fault doesn't depend on xdebug. The error occured when phpunit call testMultipleExecution. I'll try to fix it

Reply:
I finally fixed it.